src/components/userGroups/UserGroup.tsx

```diff
@@ -636,6 +637,16 @@ const CollapsedUserOrGroup: React.FC<CollapsedUserOrGroupProps> = ({
         updateCallback(index, data)
     }
 
+    function getToolTipContent(user) {
+        switch (user) {
+            case UserLocked.ADMIN :
+                return 'Admin user cannot be edited'
```

Comment:
Can we extract these messaging from constantMessaging file

Reply:
implemented 👍